unlock-app/next.config.js

```diff
@@ -32,7 +32,7 @@ Object.keys(requiredConfigVariables).forEach(configVariableName => {
   }
 })
 
-module.exports = withSourceMaps({
+module.exports = withTypescript({
```

Comment:
You'll note that in TSConfig, we set `sourceMap: true`, so we don't need `withSourceMap` anymore because it's handled by the TypeScript compiler. We do need `withTypescript` though, that's what hooks all the Next stuff up so we can just call `next build` and so on.